src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.OperationDaoImpl;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final String FILE_NAME = "src/main/resources/allTransactionForPeriod.csv";
+    private static final int INDEX_FIRST_LINE = 0;
+    private static final int INDEX_OPERATION = 0;
+    private static final int INDEX_FRUIT = 1;
+    private static final int INDEX_QUANTITY = 2;
+
+    private Transaction getFruitTransaction(String line) {
+        List<Operation> operations = new OperationDaoImpl().getListOperations();
+        String[] fields = line.split(",");
```

Comment:
It can be a constant
```suggestion
        String[] fields = line.split(SEPARATE);
```